Pyrado/scripts/training/qcp-su_sprl-multidim-all-big-ppo.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
Change file name to end with ...spdr-ppo.py (no info in there)

We do not want to have two ...spdr-ppo.py files!